packager/signer/signer.go

```diff
@@ -134,6 +134,7 @@ type Signer struct {
 	overrideBaseURL         *url.URL
 	requireHeaders          bool
 	forwardedRequestHeaders []string
+	timeNowFunc             func() time.Time
```

Comment:
(opt) Maybe remove "Func" from the name? It's kind of implied, as a non-func Time couldn't be "now".

Reply:
Done.